scipy/stats/_multivariate.py

```diff
@@ -165,10 +165,12 @@ def __init__(self, M, cond=None, rcond=None, lower=True,
             raise np.linalg.LinAlgError('singular matrix')
         s_pinv = _pinv_1d(s, eps)
         U = np.multiply(u, np.sqrt(s_pinv))
+        L = np.multiply(u, np.sqrt(np.where(s > eps, s, 0)))
```

Comment:
These don't appear to be upper and lower triangular factors of the matrix. Are L and U being used in a different sense here?

Reply:
You're right. The names don't imply lower- or upper-triangularity.

Reply:
Just for the sake of future maintainers, unless there's a good reason to name it `L`, maybe something else to avoid confusion? And perhaps mention in the comment what this matrix is for e.g.  `# A @ A.T == cov`